doozer/doozerlib/cli/__main__.py

```diff
@@ -280,6 +280,8 @@ def beta_reposync(runtime, output, cachedir, arch, repo_type, names, dry_run):
                 continue
 
             color_print('Syncing repo {}'.format(repo.name), 'blue')
+            if arch == "x86_64":
```

Comment:
Could you explain why noarch rpms should only be included in x86_64 repos? I think repos for all arches should include noarch rpms as well.

Reply:
updated